app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -75,6 +75,8 @@ def authenticate(login, password)
     logger.debug "Retrieved LDAP Attributes for #{login}: #{attrs}"
 
     attrs
+  rescue Net::LDAP::Error => error
+    raise ::Foreman::LdapException.new(N_("Error connection to %{name} LDAP server with url: %{url} during authentication: %{error}" % {:url => self.host, :name => self.name, :error => error}))
   rescue Net::LDAP::LdapError => error
```

Comment:
this is how it worked with older `net-ldap` version, you improved the error message and if we keep this for compatibility with older net-ldap, I think we can combine both exceptions here with same rescue block

long story short:
```
rescue Net::LDAP::Error, Net::LDAP::LdapError => error
  raise ::Foreman::LdapException.new(error, N_("Error while connecting to '%{name}' LDAP server at '%{url}' during authentication" % {:url => self.host, :name => self.name}))
end
```

should do the trick